AdapterConfig.json

```diff
@@ -0,0 +1,400 @@
+{
+    "version" : "0.0.0",
+    "adapters" : [
+        {
+            "id" : "adcolony",
+            "name" : "AdColony",
+            "android" : {
+                "name" : "chartboost-mediation-adapter-adcolony",
+                "sdk" : "com.adcolony:sdk",
+                "latest" : "4.8.0",
+                "versions" : [
+                    "4.8.0"
+                ]
+            },
+            "ios" : {
+                "name" : "ChartboostMediationAdapterAdColony",
+                "sdk" : "AdColony",
+                "latest" : "4.9.0",
+                "versions" : [
+                    "4.9.0"
+                ]
+            }
+        },
+        {
+            "id" : "admob",
+            "name" : "AdMob",
+            "android" : {
+                "name" : "chartboost-mediation-adapter-admob",
+                "sdk" : "com.google.android.gms:play-services-ads",
+                "latest" : "21.5.0",
+                "versions" : [
```

Comment:
Is this JSON Schema going to be manually updated and will the latest versions always be the first element in the arrays?